Example/ViewController.swift

```diff
@@ -144,6 +144,7 @@ class ViewController: UIViewController {
         clearMap.isHidden = true
         longPressHintView.isHidden = false
         
+        // TODO: adjust camera to location after cancle
```

Comment:
Do we really need this TODO statement at this point, or it was already fixed (I have a feeling that camera will adapt to the new location if location update was sent)?